db/migrate/20150320113702_add_state_to_proof_attempts_and_theorems.rb

```diff
@@ -0,0 +1,11 @@
+class AddStateToProofAttemptsAndTheorems < ActiveRecord::Migration
+  def change
+    add_column :proof_attempts, :state, :string, default: 'pending', null: false
+    add_column :proof_attempts, :state_updated_at, :datetime
+    add_column :proof_attempts, :last_error, :text
+
+    add_column :sentences, :state, :string, null: true # null for axioms
+    add_column :sentences, :state_updated_at, :datetime
+    add_column :sentences, :last_error, :text
+  end
+end
```

Comment:
When I explicitly specify `null: true` it's an unusual place for a null value and I want to emphasize that null is allowed. In this case it is only supposed to be null for axioms.